docs/notes/notes.rst

```diff
@@ -30,6 +30,10 @@ This patch release includes the following **fixes**:
   When upgrading to version 2.13.2 it is **advisable** to regenerate generated source from IDL files
    using `Fast DDS-Gen v3.2.1 <https://github.com/eProsima/Fast-DDS-Gen/releases/tag/v3.2.1>`_.
 
+.. seealso::
+  For further information about this version dependencies and other products related versions, please check the Fast
+  DDS :ref:`related_versions` section.
```

Comment:
If the suggestion  about the name of the `related_versions` section is applied, please change also here. Moreover, if we change the name we can refer directly to the specific section inside the page, i.e. `Build time dependencies`